client/pages/result.jsx

```diff
@@ -52,18 +54,18 @@ export default class Result extends React.Component {
     const { params } = parseRoute(window.location.hash);
     const term = params.get('term');
     const location = params.get('location');
-    const body = {
+    const req = {
       method: 'GET'
     };
```

Comment:
The default fetch request method is `GET` so you can leave out this `req` altogether.